cmd/capibmadm/cmd/vpc/vpc.go

```diff
@@ -35,6 +36,7 @@ func Commands() *cobra.Command {
 	cmd.PersistentFlags().StringVar(&options.GlobalOptions.ResourceGroupName, "resource-group-name", options.GlobalOptions.ResourceGroupName, "IBM cloud resource group name")
 
 	_ = cmd.MarkPersistentFlagRequired("region")
+	cmd.AddCommand(key.Commands())
 
 	cmd.AddCommand(image.Commands())
```

Comment:
nit: can we group the commands together?
```suggestion

	cmd.AddCommand(key.Commands())
	cmd.AddCommand(image.Commands())
```

Reply:
Done modified it .